cornichon-core/src/main/scala/com/github/agourlay/cornichon/core/Session.scala

```diff
@@ -75,7 +75,7 @@ case class Session(private val content: Map[String, Vector[String]]) extends Any
       }
   }
 
-  def addValues(tuples: Seq[(String, String)]) = tuples.foldLeft(this)((s, t) ⇒ s.addValue(t._1, t._2))
+  def addValues(tuples: (String, String)*) = tuples.seq.foldLeft(this)((s, t) ⇒ s.addValue(t._1, t._2))
```

Comment:
sry my bad. here is what `seq` does http://www.scala-lang.org/api/2.12.0/scala/collection/Seq.html#seq:Seq[A]
actually it is not needed. i falsely thought it was needed to compile.
`tuples` is already a Seq.
im curious though what happens if i pass `Seq("a" -> "b"): _*` to this method. it would just make sense to me to skip the call to `Seq.apply()`.